### DISCUSSION TITLE
Science-and-Technology-Society-Use-of-NASA-STELLA-Q2-Spectrometer

### DISCUSSION THREAD

**Discussion by Landsat-Mike** [STELLA-Landsat/STELLA#50] [Use Case Collection]
The **Science and Technology Society (STS) of Sarasota-Manatee Counties, Florida** is working with the NASA STELLA (Science and Technology Education for Land/Life Assessment) outreach program as a part of our STEM initiative. According to their site,

"NASA STELLA instruments are portable low-cost do-it-yourself (DIY) instruments that support science education, and outreach through scientific engagement, inquiry, and discovery while helping you understand Landsat better".

**STELLA instruments are developed under the influence and inspiration of Landsat.** This alignment not only fulfills our project needs but also serves as a compelling addition to our STEAM initiatives:

To train the minds young Floridians to be more aware of our wetlands, to care for them and about them. Our program will bring more community publicity to the issue of wetlands change, as well.

To expose our middle- and high- school aged students to real science, using real data. That means how to use instrumentation and understand how the data is collected, and how the data can be used in the real world. It means not only to create beautiful charts and images that form the good results, but also to understand that data must be collected in a proper and reproducible way, that there physics reasons for lack of accuracy and lack of precision that one must understand and minimize in order to achieve meaningful results.

Read More at:
[https://github.com/Philliec459/Science-and-Technology-Society-Use-of-NASA-STELLA-Q2-Spectrometer](https://github.com/Philliec459/Science-and-Technology-Society-Use-of-NASA-STELLA-Q2-Spectrometer)

**Comment by Huitzimengari** [comment]
Hi, we built a STELLA Q2 in the classroom and is working perfectly!!!

Greetings from Mexico!!!

Huitzi

![IMG_1738](https://github.com/STELLA-Landsat/STELLA/assets/53545729/8a1accc7-8c13-483f-986d-efc627d48558)
![IMG_1757](https://github.com/STELLA-Landsat/STELLA/assets/53545729/e55bc3b9-44d5-45f5-8586-fbf8e0eed3de)
![spectralsignature](https://github.com/STELLA-Landsat/STELLA/assets/53545729/d6d03cbf-e707-4ca1-87f3-0834166e076a)

**Reply by Landsat-Mike** [reply]
We are so happy that the STELLA-Q2 is working well for you! The work done with it is fantastic! 

Could you please email me at STELLA@mail.nasa.gov? I'd like to know more about what you and your class are doing.

**Comment by paulmirel** [comment]
Excellente!! El análisis de los datos está fabuloso!
